executor/coprocessor.go

```diff
@@ -0,0 +1,161 @@
+package executor
```

Comment:
Please add a license header for this file.

Reply:
done.